### DISCUSSION TITLE
Option to Disable Glance Animations

### DISCUSSION THREAD

**Discussion by tedbergstrand** [zen-browser/desktop#2556] [Ideas]
In real life, I've never glanced at something and waited to see it. I don't want to be forced to wait for animation transitions to snap a look into my Zen Glance.

I know some people like it to be pretty-- and I want them to have that option. But waiting on loads isn't Zen to me.

"Zen is built with performance in mind, and we have optimized the browser to be as fast as possible!"

**Comment by rodrigodagostino** [comment]
I’m right there with you, mate!

I would really like to be able to disable all transitions, it feels like a waste of time to me, especially when I open a new window. I can tell the window is already there waiting for me because I can type in the URL bar before it finishes transitioning.
